common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -92,16 +94,20 @@
     </%doc>
     <%
         from django.template import Template, Context
-        return Template("""
-            {% load render_bundle from webpack_loader %}
-            {% render_bundle entry %}
-            <script type="text/javascript">
-                {% autoescape off %}{{ body }}{% endautoescape %}
-            </script>
-        """).render(Context({
-            'entry': entry,
-            'body': capture(caller.body)
-        }))
+        try:
+            return Template("""
+                {% load render_bundle from webpack_loader %}
+                {% render_bundle entry %}
+                <script type="text/javascript">
+                    {% autoescape off %}{{ body }}{% endautoescape %}
+                </script>
+            """).render(Context({
+                'entry': entry,
+                'body': capture(caller.body)
+            }))
+        except IOError as e:
+            # Don't break Mako template rendering if the bundle or webpack-stats can't be found, but log it
```

Comment:
1. I think this is a Django template and not a Mako template.
2. Could there be any other issues?  Should we look at the error and ensure it is a webpack error?
3. What if this happened for real?  Wouldn't we want an exception?  This may fix tests, but is it unsafe for the wild?